packages/commerce-sdk-react/src/hooks/useCustomerType.ts

```diff
@@ -50,10 +51,16 @@ const useCustomerType = (): useCustomerType => {
         customerType = null
     }
 
+    const uido: string | null = onClient
+        ? // eslint-disable-next-line react-hooks/rules-of-hooks
+          useLocalStorage(`uido_${config.siteId}`)
+        : auth.get('uido')
+
```

Comment:
@kevinxh @bendvc The **isb** claim in the SLAS JWT access token tells us whether the user is an ecom user or not. Would it be okay to return this value as part of the existing `useCustomerType` hook ?

Reply:
That `useCustomerType` hook has this return type:

```
export type CustomerType = null | 'guest' | 'registered'
```

Are you going to change that? what would the return type look like?

Reply:
My gut is telling me that this right. First, the name of this type if probably not the best.. but imagine the type is called "CustomerType" ... you can see that it doesn't make sense to have anything but "type" in it. Is seems like we are sneaking uido into this hook. Maybe we can talk about it a little more.